scheduler_test.go

```diff
@@ -395,6 +395,71 @@ func TestWeekdayAt(t *testing.T) {
 	})
 }
 
+func TestDaylightSavingsScheduled(t *testing.T) {
+	loc, err := time.LoadLocation("US/Pacific")
+	assert.NoError(t, err)
+
+	fromRFC3339 := func(ts string) time.Time {
+		dt, err := time.Parse(time.RFC3339, ts)
+		assert.NoError(t, err)
+		return dt.In(loc)
+	}
+
+	before_to_edt := fakeTime{onNow: func(l *time.Location) time.Time {
```

Comment:
could you update the vars to follow the Go camelCase preference instead of snake_case? Thanks!
```suggestion
	beforeToEDT := fakeTime{onNow: func(l *time.Location) time.Time {
```

Reply:
Sorry about that, should be fixed now